HW1/P6/P6.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from functools import partial
+import findspark
+findspark.init()
+import pyspark
+
+
+def pre_process_data( filename, sc ):
+    wlist = sc.textFile( filename )
+    wlist_words = wlist.flatMap( lambda x: x.split() )
+    wlist_words_filter = wlist_words.filter( lambda x: not( x.isdigit() or x.isupper() or ( x[:-1].isupper() and x[-1:] == "." ) ) )
```

Comment:
One option to deal with this could be to check `isupper` on the result of stripping all non-alpha characters from the string.